train_management/models/train.py

```diff
@@ -32,6 +32,19 @@ def set_composition(self, vehicles):
         for sorting, vehicle in enumerate(vehicles):
             TrainConfiguration(train=self, vehicle=vehicle, sorting=sorting).save()
 
+    def add_traintimetable(self, timetables):
+
+        for timetable in timetables:
+            template = TrainTimetableTemplate.objects.get(pk=timetable.pk)
+            TrainTimetable(train=self,
+                           label=template.label,
+                           start_station=template.start_station,
```

Comment:
It might be possible to simply unpack the template object but I'm not sure if they are serializable objects and it it matters if there are too many arguments on creating the new Model. If it was possible it would be much cleaner and look something like this:
```
TrainTimetable(train=self, **template).save()
```